homeassistant/components/flunearyou/sensor.py

```diff
@@ -38,20 +38,65 @@
 SENSOR_TYPE_USER_SYMPTOMS = "symptoms"
 SENSOR_TYPE_USER_TOTAL = "total"
 
-CDC_SENSORS = [
-    (SENSOR_TYPE_CDC_LEVEL, "CDC Level", "mdi:biohazard", None),
-    (SENSOR_TYPE_CDC_LEVEL2, "CDC Level 2", "mdi:biohazard", None),
-]
-
-USER_SENSORS = [
-    (SENSOR_TYPE_USER_CHICK, "Avian Flu Symptoms", "mdi:alert", "reports"),
-    (SENSOR_TYPE_USER_DENGUE, "Dengue Fever Symptoms", "mdi:alert", "reports"),
-    (SENSOR_TYPE_USER_FLU, "Flu Symptoms", "mdi:alert", "reports"),
-    (SENSOR_TYPE_USER_LEPTO, "Leptospirosis Symptoms", "mdi:alert", "reports"),
-    (SENSOR_TYPE_USER_NO_SYMPTOMS, "No Symptoms", "mdi:alert", "reports"),
-    (SENSOR_TYPE_USER_SYMPTOMS, "Flu-like Symptoms", "mdi:alert", "reports"),
-    (SENSOR_TYPE_USER_TOTAL, "Total Symptoms", "mdi:alert", "reports"),
-]
+CDC_SENSOR_DESCRIPTIONS = (
+    SensorEntityDescription(
+        key=SENSOR_TYPE_CDC_LEVEL,
+        name="CDC Level",
+        icon="mdi:biohazard",
+        native_unit_of_measurement=None,
+    ),
+    SensorEntityDescription(
+        key=SENSOR_TYPE_CDC_LEVEL2,
+        name="CDC Level 2",
+        icon="mdi:biohazard",
+        native_unit_of_measurement=None,
+    ),
+)
+
+USER_SENSOR_DESCRIPTIONS = (
```

Comment:
```suggestion
USER_SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (
```

Reply:
Why? `mypy` doesn't complain without this, which means it can infer the type.